src/firmware/app/control/trajectory_planner.c

```diff
@@ -164,11 +167,13 @@ void app_trajectory_planner_createForwardsContinuousAngularSpeedProfile(
 
     for (unsigned int i = 1; i < num_segments; i++)
     {
+        const float speed        = trajectory->trajectory_elements[i - 1].angular_speed;
+        const float displacement = segment_lengths[i - 1].angular_segment_length;
+
         // Vf = sqrt( Vi^2 + 2*constant_segment_length*max_acceleration)
```

Comment:
nit: move ` // Vf = sqrt( Vi^2 + 2*constant_segment_length*max_acceleration)` to the physics file

Reply:
Moved 👍